Remotion/Web/Core/Res/Themes/NovaViso/HTML/FormGrid.css

```diff
@@ -1,105 +1,84 @@
 /*  Style applied to the form grid table's outer structure  */
-table.formGridTable
+.formGridTable
 {
   border-collapse: collapse;
-  border-width: 0;
   width: 100%;
+  background-color: var(--color-standard-background);
 }
 
-td.formGridTitleCell,
-td.formGridSubTitleCell
+.formGridTitleCell,
+.formGridSubTitleCell
 {
   font-weight: bold;
```

Comment:
I've looked around and I think I found a good example for how the titlecell should look:
![image](https://user-images.githubusercontent.com/6972548/132809154-1623401e-208e-49ec-ac92-81a5fadfa9d8.png)

Also, we need a configurable padding, settable via a css variable so the formgrid can be displayed with and without a padding because we can't do the gray background otherwise. The default is probably going to be padding, but it can be overridden to zero via CSS